core/assets/bundles/bundle_vi.properties

```diff
@@ -1,2044 +1,2044 @@
-block.reinforced-liquid-router.name = Reinforced Liquid Router
-block.reinforced-liquid-container.name = Reinforced Liquid Container
-block.reinforced-liquid-tank.name = Reinforced Liquid Tank
-block.beam-node.name = Beam Node
-block.beam-tower.name = Beam Tower
-block.beam-link.name = Beam Link
-block.turbine-condenser.name = Turbine Condenser
-block.chemical-combustion-chamber.name = Chemical Combustion Chamber
-block.pyrolysis-generator.name = Pyrolysis Generator
-block.vent-condenser.name = Vent Condenser
-block.cliff-crusher.name = Cliff Crusher
-block.plasma-bore.name = Plasma Bore
-block.large-plasma-bore.name = Large Plasma Bore
-block.impact-drill.name = Impact Drill
-block.eruption-drill.name = Eruption Drill
-block.core-bastion.name = Core Bastion
-block.core-citadel.name = Core Citadel
-block.core-acropolis.name = Core Acropolis
-block.reinforced-container.name = Reinforced Container
-block.reinforced-vault.name = Reinforced Vault
-block.breach.name = Breach
-block.sublimate.name = Sublimate
-block.titan.name = Titan
-block.disperse.name = Disperse
-block.fabricator.name = Fabricator
-block.tank-reconstructor.name = Tank Reconstructor
-block.mech-reconstructor.name = Mech Reconstructor
-block.ship-reconstructor.name = Ship Reconstructor
-block.tank-assembler.name = Tank Assembler
-block.ship-assembler.name = Ship Assembler
-block.mech-assembler.name = Mech Assembler
-block.reinforced-payload-conveyor.name = Reinforced Payload Conveyor
-block.reinforced-payload-router.name = Reinforced Payload Router
-block.payload-mass-driver.name = Payload Mass Driver
-block.small-deconstructor.name = Small Deconstructor
-block.canvas.name = Canvas
-block.world-processor.name = World Processor
-block.world-cell.name = World Cell
-block.shield-breaker.name = Shield Breaker (temp name/sprite)
-block.tank-fabricator.name = Tank Fabricator
-block.mech-fabricator.name = Mech Fabricator
-block.ship-fabricator.name = Ship Fabricator
-block.refabricator.name = Refabricator
-block.prime-refabricator.name = Prime Refabricator
-block.unit-repair-tower.name = Unit Repair Tower
-block.diffuse.name = Diffuse
-block.basic-assembler-module.name = Basic Assembler Module
-unit.stell.name = Stell
-unit.locus.name = Locus
-unit.vanquish.name = Vanquish
-unit.conquer.name = Conquer
-unit.cleroi.name = Cleroi
-unit.tecta.name = Tecta
-unit.collaris.name = Collaris
-unit.avert.name = Avert
-unit.quell.name = Quell
-unit.disrupt.name = Disrupt
-unit.evoke.name = Evoke
-unit.incite.name = Incite
-unit.emanate.name = Emanate
-unit.manifold.name = Manifold
-unit.assembly-drone.name = Assembly Drone
-unit.effect-drone.name = Effect Drone
-unit.precept.name = Precept
-unit.merui.name = Merui
-unit.anthicus.name = Anthicus
-unit.elude.name = Elude
-unit.obviate.name = Obviate
-sector.two.name = Two
-sector.three.name = Three
-sector.four.name = Four
-sector.five.name = Five
+credits.text = Được tạo ra bởi [royal]Anuken[] - [sky]anukendev@gmail.com[]

```

Comment:
What is that character ????

Reply:
Please reformat the file return back to normal bruh

Reply:
I dont know how on mobile

Reply:
Could you do that???

Reply:
I has invited you to be a moderator of patch-1
Pls accept it and keep translating because i will be busy till this weekend

Reply:
I will open another PR for my update

Reply:
Ok

Reply:
Could I close this pr?

Reply:
Could we communicate on FB???